__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const app = require("../db/app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("200: should get all topics as an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.topics)).toBe(true);
+        expect(body.topics.length).toBe(3);
+      });
+  });
+  test("200: each topic should have slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
```

Comment:
lets test the length before the forEach in the same test pls :) you can condense these into one test